R/declare_estimator.R

```diff
@@ -244,8 +244,10 @@ tidy_estimator <- function(estimator_function) {
 
 #' @param data a data.frame
 #' @param model A model function, e.g. lm or glm. By default, the model is the \code{\link{difference_in_means}} function from the \link{estimatr} package.
+#' @param post_estimation A model-in data-out function to extract coefficient estimates or model summary statistics, such as \code{\link{tidy}} or \code{\link{glance}}. By default, the \code{DeclareDesign} post-estimation function \code{\link{tidy_filter}} is used, which tidies data and optionally filters to relevant coefficients.
```

Comment:
Add a note to the docs that term is threaded through to post_estimation, it can be hard to trace which function gets called with what.